testsuite/MDAnalysisTests/transformations/test_nojump.py

```diff
@@ -0,0 +1,49 @@
+###
+
+import numpy as np
+import pytest
+from numpy.testing import assert_array_almost_equal
+
+import MDAnalysis as md
```

Comment:
We traditionally import as mda, please change for consistency

Reply:
Done